src/cli/analytics.rs

```diff
@@ -77,51 +86,148 @@ impl Command for Analytics {
     }
 }
 
+pub struct AnalyticsStream {
+    span: Span,
+    streams: StreamMap<String, RawJsonRowStreamer>,
+    rt: Arc<Runtime>,
```

Comment:
This field never gets read but is required as otherwise the runtime is closed at the end of the function before we can read from the streams that were created in it. This was the runtime's lifetime is the same as the streams